security/clamav/src/opnsense/mvc/app/controllers/OPNsense/ClamAV/forms/general.xml

```diff
@@ -0,0 +1,186 @@
+        <label>Scan PDF</label>
+        <type>checkbox</type>
+        <help>This option enables scanning within PDF files.</help>
+    </field>
+    <field>
+        <id>general.scanswf</id>
+        <label>Scan SWF</label>
+        <type>checkbox</type>
+        <help>This option enables scanning within SWF files.</help>
+    </field>
+    <field>
+        <id>general.scanxmldocs</id>
+        <label>Scan XMLDOCS</label>
+        <type>checkbox</type>
+        <help>This option enables scanning xml-based document files supported by libclamav.</help>
+    </field>
+    <field>
+        <id>general.scanhwp3</id>
+        <label>Scan HWP3</label>
+        <type>checkbox</type>
+        <help>This option enables scanning of HWP3 files.</help>
+    </field>
+    <field>
+        <id>general.scanmailfiles</id>
+        <label>Scan mail files</label>
+        <type>checkbox</type>
+        <help>Enable internal e-mail scanner with all defaults enabled. If you turn off this option, the original files will still be scanned, but without parsing individual messages/attachments.</help>
+    </field>
+    <field>
+        <id>general.scanhtml</id>
+        <label>Scan HTML</label>
+        <type>checkbox</type>
+        <help>Perform HTML normalisation and decryption of MS Script Encoder code.</help>
+    </field>
+    <field>
+        <id>general.scanarchive</id>
+        <label>Scan archives</label>
+        <type>checkbox</type>
+        <help>ClamAV will scan within archives and compressed files.</help>
+    </field>
+    <field>
+        <id>general.arcblockenc</id>
+        <label>Block encrypted archive</label>
+        <type>checkbox</type>
+        <help>Mark encrypted archives as viruses (Encrypted.Zip, Encrypted.RAR).</help>
+    </field>
+    <field>
+        <id>general.maxscansize</id>
+        <label>Max scan size</label>
+        <type>text</type>
+        <help>This option sets the maximum amount of data to be scanned for each input file. Archives and other containers are recursively extracted and scanned up to this value.</help>
+    </field>
+     <field>
+        <id>general.maxfilesize</id>
+        <label>Max file size</label>
+        <type>text</type>
+        <help>Files larger than this limit won't be scanned.</help>
+    </field>
+     <field>
+        <id>general.maxrecursion</id>
+        <label>Max recursion</label>
+        <type>text</type>
+        <help>Nested archives are scanned recursively, e.g. if a Zip archive contains a RAR file, all files within it will also be scanned.</help>
+    </field>
+     <field>
+        <id>general.maxfiles</id>
+        <label>Max files</label>
+        <type>text</type>
+        <help>Number of files to be scanned within an archive, a document, or any other container file.</help>
+    </field>
+    <field>
+        <id>general.fc_logtime</id>
+        <label>Freshclam log time</label>
+        <type>checkbox</type>
+        <help>Log time with each message.</help>
```

Comment:
thats probably a better solution as you probably need a format that never change. The worst case: The user disables it and enables it later and the new log lines get appended to the old one. Then it will look quite strange (or crash if not well written).